lib/index.js

```diff
@@ -69,6 +69,13 @@ internals.Connection.prototype.start = function (callback) {
 
 internals.Connection.prototype.stop = function () {
 
+    // Clean up pending eviction timers.
+    for (var segment in this.cache) {
```

Comment:
Can you please change both of these `for...in` loops to iterate over `Object.keys()`.